### PR TITLE
xcodegen 0.0.2

### PR DESCRIPTION
**Pull request by brandtdaniels**
TagCheck

### New Pull Request Checklist

*Please mark the points which you did / accept.*

- [x] __I will not move an already shared step version's tag to another commit__
- [x] I read the [Step Development Guideline](https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md)
- [x] I have a test for my Step, which can be run with `bitrise run test` (in the step's repository)
- [x] I did run `bitrise run audit-this-step` (in the step's repository - note, if you don't have this workflow in your `bitrise.yml`, [you can copy it from the step template](https://github.com/bitrise-steplib/step-template/blob/master/bitrise.yml).)
- [x] I read and accept the [Abandoned Step policy](https://github.com/bitrise-io/bitrise-steplib#abandoned-step-policy)

### CODE REVIEW

File: steps/xcodegen/0.0.2/step.yml [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+website: https://github.com/brandtdaniels/xcodegen-bitrise
+source_code_url: https://github.com/brandtdaniels/xcodegen-bitrise
+support_url: https://github.com/brandtdaniels/xcodegen-bitrise/issues
+published_at: 2019-10-15T15:34:57.183896-07:00
+source:
+  git: https://github.com/brandtdaniels/xcodegen-bitrise.git
+  commit: 3457781eb072e5673ddfa3a35806edc21eda8236
+host_os_tags:
+- osx-10.10
+- ubuntu-16.04
```
Comment (yonaskolb) [inline steps/xcodegen/0.0.2/step.yml]: XcodeGen currently doesn’t run on Linux

File: steps/xcodegen/0.0.2/step.yml [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+project_type_tags:
+- ios
+- macos
```
Comment (yonaskolb) [inline steps/xcodegen/0.0.2/step.yml]: Works for tvos and watchos as well. Don’t know if they are specific types in bitrise

File: steps/xcodegen/0.0.2/step.yml (hunk 1 above)
Comment (brandtdaniels) [reply steps/xcodegen/0.0.2/step.yml]: Looks like it's currently unused so it shouldn't be an issue

https://devcenter.bitrise.io/jp/bitrise-cli/step-properties/

> host_os_tags : supported operating systems. Currently unused, reserved for future use.

File: steps/xcodegen/0.0.2/step.yml (hunk 2 above)
Comment (brandtdaniels) [reply steps/xcodegen/0.0.2/step.yml]: These are optional and only used for filtering

> project_type_tags : project type tags if the Step is project type specific. Example: ios or android. Completely optional, and only used for search and filtering in Step lists.

**Comment by toshi0383** [comment]
Nice😍
Is XcodeGen’s version pinned or the latest by default?

**Comment by brandtdaniels** [comment]
I believe it would install the latest

**Comment by brandtdaniels** [comment]
So I'm new to this bitrise step process...How do I update a PR? Or do I just run `bitirse run share-this-step` again

**Comment by BirmacherAkos** [comment]
Hi @brandtdaniels 

Sorry for the late response.
After you did the changes you have to create a new release of your step (0.0.3) and share the new version with the fix.

**Comment by brandtdaniels** [comment]
@BirmacherAkos I've pushed up a new tag. Do I need to create a new PR?

**Comment by BirmacherAkos** [comment]
@brandtdaniels Yes please do a new one.
The step.yml in this PR is locked on the 0.0.2 version so you will need a new one for the 0.0.3 version.

**Comment by godrei** [comment]
@brandtdaniels closing this PR as new version was submitted

**Pull request closed without merging**